processing/src/main/java/org/apache/carbondata/processing/newflow/steps/SortProcessorStepImpl.java

```diff
@@ -50,6 +50,7 @@ public SortProcessorStepImpl(CarbonDataLoadConfiguration configuration,
 
   @Override
   public void initialize() throws CarbonDataLoadingException {
+    super.initialize();
```

Comment:
This is a bit confusing, I think it is better each step to initialize its child. So you can make initialize function as abstract in the base class